src/components/Link.md

```diff
@@ -0,0 +1,6 @@
+```jsx
+<div>
+  <Link href="/">Normal link</Link>{' '}
+  <Link href="/" shy>Shy link</Link>
+</div>
+```
```

Comment:
It would be cool to have a little description with each component. For example, I didn't find it self explanatory what the “shy” prop does.

Reply:
Yeah, that's planned ;-)

Reply:
Done ;-)